internal/tui/templates/templater.go

```diff
@@ -34,6 +34,12 @@ func formatCommand(name string, desc string, padding int) string {
 	return fmt.Sprintf("  %s    %s", styledName, styledDesc)
 }
 
+var customHelpShortMessage = map[string]string{
+	"help": "Display help information for Atmos commands",
+	"tf":   "Execute Terraform commands (e.g., plan, apply, destroy) using Atmos stack configurations",
+	"show": "Display help for show",
```

Comment:
I don't know that we have a show command.

Reply:
Now even I am curious. I found it but now even I don't see it.